### PR TITLE
Send Metrics Home

### PR DESCRIPTION
**Pull request by eliobischof**

### CODE REVIEW

**Comment by eliobischof** [comment]
Only rules are still open

**Comment by thesephirot** [comment]
it would be cool, if rules could be applied as an additional (or additive) configmap to be able to adjust on demand.

**Comment by eliobischof** [comment]
> it would be cool, if rules could be applied as an additional (or additive) configmap to be able to adjust on demand.

I think our rule changes should apply for all installations, therefore they should be released with boom itself. If a customer needs different rules, they can always deploy them manually or via pre- or postapply. Other opinions?

> As I see the labels on boom and orbiter have changed with app.kubernetes.io/part-of: orbos, shouldn't the labels on the from boom deployed application change as well?

Good point, I'll do it.

> As the metrics are available, the dashboards should be added to the /dashboards folder as well

I'll go for it

**Comment by fforootd** [comment]
> I think our rule changes should apply for all installations, therefore they should be released with boom itself. If a customer needs different rules, they can always deploy them manually or via pre- or postapply. Other opinions?

👍

**Comment by thesephirot** [comment]
hmm,.. had another thought,.. rules can be applied without the actual scraper, fine.

would you then apply the blackbox ones as well please:

```
apiVersion: monitoring.coreos.com/v1
kind: PrometheusRule
metadata:
  name: caos-blackexporter-rules
  namespace: caos-system
  labels:
    boom.caos.ch/prometheus: caos
spec:
  groups:
   - name: caos.rules
     rules:
        - expr: min (probe_success) by (instance)
          record: caos_blackboxexporter_probesuccess
        - expr: min by (instance) (probe_ssl_earliest_cert_expiry-time())
          record: caos_blackboxexporter_ssl_expiry
        - expr: min by (instance) (probe_http_ssl)
          record: caos_blackboxexporter_ssl_termination
        - expr: min by (instance)  (probe_http_status_code())
          record: caos_blackboxexporter_http_statuscode
```

File: scripts/3_remotewrite-secret.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# getter for gopass and secret yaml creation
+set -e
+[[ `uname` = "Linux" ]] && ENCODE="base64 --wrap=0" || ENCODE="base64"
+
+cat <<EOL
+apiVersion: v1
+data:
+  username: $(gopass abraxas-secrets/aap/grafana-cloud/user | $ENCODE)
+  password: $(gopass abraxas-secrets/aap/grafana-cloud/apikey | $ENCODE)
```
Comment (fforootd) [inline scripts/3_remotewrite-secret.sh:10]: please remove or change these

**Pull request merged**